.github/workflows/cd.yml

```diff
@@ -64,8 +64,10 @@ jobs:
 
     - name: Add final tags
       run: |
-        docker pull ghcr.io/vmware/repository-service-tuf-api:${{ github.ref_name }}
-        docker tag ghcr.io/vmware/repository-service-tuf-api:${{ github.ref_name }} latest
+        docker pull ghcr.io/vmware/repository-service-tuf-api:${{ github.sha }}
```

Comment:
Why did you need to pull the image from the GitHub container registry?
Don't you have to build it locally?

Reply:
It was build in another job by another workflow.

Reply:
If you mean in the `publish_container.,yml` workflow I want to point out that there the default `image_version` is `dev` and not `github.sha` meaning we are using `dev` and `github.sha` together.
If I understand the github documentation well enough they should be the same which is fine.